sources/packages/web/src/components/common/AssessmentAwardDetails.vue

```diff
@@ -387,6 +478,34 @@ export default defineComponent({
       return undefined;
     };
 
+    const canCancelFirstDisbursement = computed<boolean>(() => {
+      debugger;
+      return (
+        props.allowDisbursementCancellation &&
+        props.assessmentAwardData.estimatedAward["disbursement1Status"] ===
+          DisbursementScheduleStatus.Sent &&
+        !receivedDisbursementReceipt(
+          "first",
+          props.assessmentAwardData.finalAward,
+        )
+      );
```

Comment:
Let mw know if I am missing something. Would it be more simple to use the computed like this. 

```ts

return (
        props.allowDisbursementCancellation &&
        props.assessmentAwardData.estimatedAward.disbursement1Status ===
          DisbursementScheduleStatus.Sent &&
        !props.assessmentAwardData.finalAward?.disbursementReceipt1Received
      );
```

Reply:
I will change it. These properties were consumed as "dynamic" in the code, but they are not actually dynamic, nor are they expressed as properties in the DTO (which they should). Since they are already directly accessed in the template, here should be ok too.

Reply:
Changed.